astropy/coordinates/tests/test_angles.py

```diff
@@ -93,34 +91,32 @@ def test_create_angles():
     assert_allclose(a5.radian, a6.radian)
 
     assert_allclose(a10.degree, a11.degree)
-    assert a11 == a12 == a13 == a14
+    assert a11 == a14
     assert a21 == a22
     assert a23 == -a24
     assert a24 == a25
 
     # check for illegal ranges / values
     with pytest.raises(IllegalSecondError):
-        a = Angle("12 32 99", unit=u.degree)
+        Angle("12 32 99", unit=u.degree)
```

Comment:
It is a refactoring that will never get backported, so I don't see a big deal. I can undo these changes but I probably won't come back to do just this clean up.

Reply:
I'm happy for you to leave them in, since you've done them. A new PR is enough of a pain that sometimes one just fixes up little things in unrelated ones... (well, I certainly do, even though I sometimes complain about it too!!)

Reply:
Opening a separate PR for this should not be necessary, but I'd rather move these to a separate commit myself than keep everything in a single commit.

Reply:
> I'd rather move these to a separate commit myself than keep everything in a single commit.

That is even more work for me than just removing them. It is Friday and I want to move this off my todo. Should I just remove the changes you don't like even though you agree they are good changes?

Reply:
For many of these tests the best thing to do (but clearly beyond the scope of this pull request) would be to rewrite them as parametrized tests. For doing that the changes in question will not be important and applying them would create code churn. So if you want to undo these changes then that's fine with me.

Reply:
@eerovaher - I think you maybe take @pllim too literally: we all agree these changes are good, but differ on whether it is worth the *work* and CI time to split things up in separate PRs or commits just so the commit history is a bit cleaner. @pllim wants to be done (and I'd rather she have as much time as possible to continue to keep things afloat!), and feels this is not worth more work, while you seem to feel differently. While I like how you have made astropy better in having single, well-contained commits and PRs, here to me the effort-to-value ratio (in both work and CI time and thus CO2) really seems rather low. So, I suggest we just merge this. Or maybe you split how you like it and force-push? I'm sure @pllim would be OK with that (we've done that mutually often enough, just trying to optimize time spent).